storage/gcloud/aio/storage/storage.py

```diff
@@ -302,6 +302,38 @@ async def upload_from_filename(self, bucket: str, object_name: str,
             return await self.upload(bucket, object_name, file_object,
                                      **kwargs)
 
+    async def set_temporary_hold(
+            self, bucket: str, object_name: str, *, hold: bool = True,
+            params: Optional[Dict[str, str]] = None,
+            session: Optional[Session] = None, timeout: int = 10
+    ) -> Dict[str, Any]:
+
```

Comment:
Please toss a comment linking to https://cloud.google.com/storage/docs/json_api/v1/objects/patch here

Reply:
(added to the private method instead.)